### PR TITLE
Manage sponsorship attempts after reaching limits of 20 sponsors

### PR DESCRIPTION
**Pull request by mybits**
different scenarios described here:
PT: https://www.pivotaltracker.com/n/projects/307301/stories/94792936

Added one commit to fix signature route after rebasing

Added new commit after updating ACs - we want also stop collecting sponsors on petitions that are still pending (no one has validated its email)

### CODE REVIEW

File: spec/controllers/signatures_controller_spec.rb [hunk 1]
```diff
@@ -86,6 +86,17 @@
         end
       end
 
+      it 'doesn\'t send email notification to the petition creator if the petition is already in moderation' do
+        petition_in_moderation = FactoryGirl.create(:petition, state: 'sponsored')
+        sponsor = FactoryGirl.create(:sponsor, petition: petition_in_moderation)
+        signature = sponsor.create_signature(FactoryGirl.attributes_for(:pending_signature, petition: petition_in_moderation))
+        perform_enqueued_jobs do
```
Comment (otlaitil) [inline spec/controllers/signatures_controller_spec.rb]: We could probably use another approach to assert that the request does not add any jobs to the queue:  http://edgeapi.rubyonrails.org/classes/ActiveJob/TestHelper.html#method-i-assert_no_enqueued_jobs
```
assert_no_performed_jobs do
  get :verify, :id => signature.id, :token => signature.perishable_token
end
```

**Comment by otlaitil** [comment]
Looks good! :+1: The test suite is very comprehensive!

File: spec/controllers/signatures_controller_spec.rb (hunk 1 above)
Comment (mybits) [inline spec/controllers/signatures_controller_spec.rb]: Oh that's neat! Will implement!

Comment (mybits) [inline spec/controllers/signatures_controller_spec.rb]: Just added it here: https://github.com/alphagov/e-petitions/commit/abc5d4bd5f63e73f9c6f0db5ec2128df5982946a
Thans for the tip!

File: app/views/petitions/moderation_info.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<% content_for(:page_title) { "#{@petition.title} - has been already sent to moderation - e-petitions" } %>
+<% content_for :title do %>
+<div class="title_block">
+  <h1>Thank you for your support, but <%= @petition.title %> petition has been already sent to moderation.</h1>
```
Comment (yahoopete) [inline app/views/petitions/moderation_info.html.erb]: Can we change the words to:

"<name of creator>'s petition "<petition title>" already has its 5 sponsors and we're taking a look to make sure it meets the Petitions service terms and conditions."

"The petition should be available for signing in a few days".

Comment (mybits) [inline app/views/petitions/moderation_info.html.erb]: Sure, changed here: https://github.com/alphagov/e-petitions/commit/426c61a16b2db7d441c2bc35ae354fb76cf23915

**Comment by davidbasalla** [comment]
looks good to me!

**Pull request merged**